### PR TITLE
Updates for Patch Tuesday

### PR DESCRIPTION
**Pull request by mthalman**

### CODE REVIEW

File: 3.5/runtime/windowsservercore-1903/Dockerfile [hunk 1]
```diff
@@ -10,6 +10,14 @@ RUN curl -fSLo microsoft-windows-netfx3.zip https://dotnetbinaries.blob.core.win
     && del microsoft-windows-netfx3-ondemand-package~31bf3856ad364e35~amd64~~.cab `
     && powershell Remove-Item -Force -Recurse ${Env:TEMP}\*
 
+# Apply latest patch
```
Comment (MichaelSimons) [inline 3.5/runtime/windowsservercore-1903/Dockerfile]: I'm not following why this is necessary.  Isn't this patch the OS LCU?  .NET for 1903 is patched separately from the OS.

Comment (mthalman) [reply 3.5/runtime/windowsservercore-1903/Dockerfile]: Ah, yep.

File: manifest.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "readmePath": "README.md",
   "registry": "mcr.microsoft.com",
   "variables": {
-    "RuntimeReleaseDateStamp": "20190514",
-    "SdkReleaseDateStamp": "20190521"
+    "RuntimeReleaseDateStamp": "20190611",
```
Comment (MichaelSimons) [inline manifest.json]: The repo readme's should be regenerated as a result of this change.

Comment (mthalman) [reply manifest.json]: Yep, just committed it.

**Pull request merged**